src/components/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+type Props = {
+  filter: Filter,
+  onSetFilter: (filter: Filter) => void,
+  completedTodos: Todo[],
+  onRemoveAll: (userId: number) => void,
+  activeTodos: Todo[];
+};
+
+export const Footer: React.FC<Props> = ({
+  filter,
+  onSetFilter,
+  completedTodos,
+  onRemoveAll,
+  activeTodos,
+}) => (
+  <footer className="todoapp__footer">
+    <span className="todo-count">
+      {`${activeTodos.length} items left`}
+    </span>
+
+    <nav className="filter">
+      <a
+        href="#/"
+        className={classNames(
+          'filter__link',
+          {
+            selected: filter === 'all',
+          },
+        )}
+        onClick={() => onSetFilter(Filter.All)}
+      >
+        All
+      </a>
+
+      <a
+        href="#/active"
+        className={classNames(
+          'filter__link',
+          {
+            selected: filter === 'active',
+          },
+        )}
+        onClick={() => onSetFilter(Filter.Active)}
+      >
+        Active
+      </a>
+
+      <a
+        href="#/completed"
+        className={classNames(
+          'filter__link',
+          {
+            selected: filter === 'completed',
+          },
+        )}
+        onClick={() => onSetFilter(Filter.Completed)}
+      >
+        Completed
+      </a>
+    </nav>
+
+    <button
+      type="button"
+      className="todoapp__clear-completed"
+      disabled={completedTodos.length === 0}
```

Comment:
Get used to using logical operators, please 
```suggestion
      disabled={!completedTodos.length}
```